backend/runner/runner.go

```diff
@@ -397,7 +406,8 @@ func (s *Service) deploy(ctx context.Context, key model.DeploymentKey, module *s
 			return fmt.Errorf("failed to download artefacts: %w", err)
 		}
 
-		envVars := []string{"FTL_ENDPOINT=" + s.config.ControllerEndpoint.String(),
+		logger.Infof("Setting FTL_ENDPOINT to %s", s.proxy.bindAddress.String())
```

Comment:
Should be debugf to avoid showing to user during local dev. We should really solve this differently, but for now that's the way it is.